pkg/services/accesscontrol/models.go

```diff
@@ -455,6 +455,7 @@ const (
 	ActionAlertingReceiversCreate           = "alert.notifications.receivers:create"
 	ActionAlertingReceiversUpdate           = "alert.notifications.receivers:write"
 	ActionAlertingReceiversDelete           = "alert.notifications.receivers:delete"
+	ActionAlertingReceiversTest             = "alert.notifications.receivers:test"
```

Comment:
I'm not sure how strict we are with this concept right now, but there will be a future where the action names are restricted on only be the standard CRUD verbs. Given that, the `test` verb would be considered non-standard in the future.

Is it something where the verb can be converted into the resource name? E.g. `alert.notifications.receivers.tests:write`.

Reply:
the background on this is that app platform built on K8S API design will be restricting how we authorize to a consistent set of verbs, the ones supported by K8S RBAC. There would be a lot of edge cases to handle, but perhaps there will be separate methods for endpoints that don't operate on a resource and want to perform more of an RPC-style operation than a REST operation.

But... maybe `datasource.queries:read` or `datasource.queries:create`?